src/bootstrap/bootstrap.py

```diff
@@ -678,7 +678,7 @@ def update_submodule(self, module, checked_out, recorded_submodules):
         run(["git", "submodule", "-q", "sync", module],
             cwd=self.rust_root, verbose=self.verbose)
         run(["git", "submodule", "update",
-            "--init", "--recursive", module],
+            "--init", "--recursive", "--progress", module],
```

Comment:
Nit: it'd be nice to use git rebase -i to remove this from the history altogether (same with the printlns)

Reply:
I would absolutely like to tidy up the git history, but I am not familiar enough with git to really know how to start. I am know about rebase, but I need to do some more learning to be able to use it to clean up a git history. I'll get right on it.

Reply:
@Goirad I'll talk to you offline